### PR TITLE
Feature/skills

### PR DESCRIPTION
**Pull request by varaMbicycle**
- resolved issue to edit name skill
- changed addTool button position
- added scroll func to tool list

### CODE REVIEW

File: src/common/providers/AppProvider.tsx [hunk 1]
```diff
@@ -14,19 +14,19 @@ import theme from 'common/theme';
...
-          <ThemeProvider theme={theme}>
-            <BrowserRouter>
-              <MsalProvider instance={msalInstance}>
-                {children}
-              </MsalProvider>
-            </BrowserRouter>
-          </ThemeProvider>
-        </LocalizationProvider>
-      </React.StrictMode>
+      {/* <React.StrictMode> */}
```
Comment (alexMbicycle) [inline src/common/providers/AppProvider.tsx]: Please delete from the code

File: src/containers/main-page/cv-form/components/fields/skills/components/tool/Skill.tsx [hunk 2]
```diff
@@ -81,10 +92,25 @@ const Skill = function ({ skill }: SkillProps): JSX.Element {
...
+        {/*   justifyContent="flex-start" */}
+        {/* > */}
+        {/*   <AddToolButtonStyled */}
+        {/*     disabled={!skill.name} */}
+        {/*     onClick={onAddToolHandle} */}
+        {/*   > */}
+        {/*     <AddCircleIconStyled /> */}
+        {/*     {Text.AddTool} */}
+        {/*   </AddToolButtonStyled> */}
+        {/* </Grid> */}
```
Comment (alexMbicycle) [inline src/containers/main-page/cv-form/components/fields/skills/components/tool/Skill.tsx]: Do not use commented code in the code.

File: src/containers/main-page/cv-form/components/fields/skills/components/tool/index.tsx [hunk 3]
```diff
@@ -24,10 +26,10 @@ const Tool = function ({
   skillId,
 }: ToolProps): JSX.Element {
   const { dispatch } = useCategoryContext();
-
   const onDeleteToolHandle = useCallback((): void => {
     dispatch({ type: 'remove-tool', tool, skillId });
   }, [dispatch, skillId, tool]);
+  const listRef = useRef<HTMLDivElement>(null);
```
Comment (alexMbicycle) [inline src/containers/main-page/cv-form/components/fields/skills/components/tool/index.tsx]: Should be placed before functions with space

**Review by alexMbicycle — CHANGES_REQUESTED** [review]

File: src/common/components/text-field-outlined/index.tsx [hunk 4]
```diff
@@ -1,5 +1,5 @@
 import { OutlinedTextFieldProps } from '@mui/material';
-import { memo } from 'react';
+import React, { memo } from 'react';
 import { TextFieldStyled } from './styled';
```
Comment (alexMbicycle) [inline src/common/components/text-field-outlined/index.tsx]: Should look like this

```
import React, { memo } from 'react';
import { OutlinedTextFieldProps } from '@mui/material';

import { TextFieldStyled } from './styled';
```

File: src/containers/main-page/cv-form/components/fields/skills/components/tool/index.tsx [hunk 5]
```diff
@@ -1,6 +1,11 @@
-import { memo, useCallback } from 'react';
+import React, {
+  memo, useCallback, useEffect, useRef, useState,
+} from 'react';
```
Comment (alexMbicycle) [inline src/containers/main-page/cv-form/components/fields/skills/components/tool/index.tsx]:
```
import React, {
  memo, useCallback, 
  useEffect, useRef, useState,
} from 'react';
```

File: src/containers/main-page/cv-form/components/fields/skills/components/tool/index.tsx [hunk 6]
```diff
@@ -1,6 +1,11 @@
...
 
-import { Grid, Typography, SelectChangeEvent } from '@mui/material';
+import {
+  Grid, Typography, SelectChangeEvent, AccordionSummary, AccordionDetails,
+} from '@mui/material';
```
Comment (alexMbicycle) [inline src/containers/main-page/cv-form/components/fields/skills/components/tool/index.tsx]:
```
import {
  Grid, Typography, SelectChangeEvent, 
  AccordionSummary, AccordionDetails,
} from '@mui/material';
```

File: src/containers/main-page/cv-form/components/fields/skills/utils/styled.ts [hunk 7]
```diff
@@ -1,5 +1,5 @@
 import {
-  Box, Button, Divider, Grid, styled, Typography,
+  Box, Button, Divider, Grid, styled, Typography, Accordion,
```
Comment (alexMbicycle) [inline src/containers/main-page/cv-form/components/fields/skills/utils/styled.ts]: please drop by three items

**Review by alexMbicycle — CHANGES_REQUESTED** [review]

File: src/containers/main-page/cv-form/components/fields/skills/utils/styled.ts (hunk 7 above)
Comment (varaMbicycle) [reply src/containers/main-page/cv-form/components/fields/skills/utils/styled.ts]: I will know for future

**Review by alexMbicycle — APPROVED** [review]

**Pull request merged**